src/dune_pkg/rev_store.ml

```diff
@@ -2,43 +2,97 @@ open Stdune
 open Dune_vcs
 module Process = Dune_engine.Process
 module Display = Dune_engine.Display
+module Scheduler = Dune_engine.Scheduler
 module Re = Dune_re
+module Flock = Dune_util.Flock
 open Fiber.O
 
-type t = { dir : Path.t }
+type t =
```

Comment:
Since `lock` and `dir` are always relative to each other I would argue to keep just `dir` in `t` and compute the lock path from it.

Reply:
Indeed, this is better. I had this left over from when the lock was an actual sort of lock but now that it is just a `Path.t` it makes no sense. Thanks for pointing it out!.